src/presentation/common/components/module-package-list-table/ModulePackageListTable.tsx

```diff
@@ -13,23 +13,32 @@ import { useViewSelector, ViewSelectorOptions } from "./useViewSelector";
 
 export interface ModulePackageListTableProps {
     onCreate?(): void;
-    onViewChange?(option: ViewOption): void;
-    viewValue?: ViewOption;
+    onViewChange?(option: ViewOptions): void;
+    viewValue?: ViewOptions;
     showSelector: ViewSelectorOptions;
     showInstances: InstanceSelectionOptions;
+    presentation: PresentationOptions;
 }
 
-export type ViewOption = "modules" | "packages";
+export type ViewOptions = "modules" | "packages";
```

Comment:
[code style] Not sure about this. Conceptually, a singular name seems more fitting (it's not a collection, it's a union of possibilities), and it looks good specially on usage: `type Color = "red" | "blue"; function getRGB(color: Color) { ... }`  Example from the web (fruit): https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-3.html

Reply:
I agree with that, however we were not consistent on the rest of types/interfaces.

For example we had ``InstanceSelectionOptions`` or ``ViewSelectorOptions``. Renamed those to ``InstanceSelectionConfig`` and ``ViewSelectorConfig``. So that we have ``FooOption`` types and ``BarConfig`` interfaces (where option does not mean an union of possibilities)